playbooks/move_vm_from_on_prem_to_gcp/move_vm_from_on_prem_to_gcp.yml

```diff
@@ -0,0 +1,44 @@
+- name: Playbook to migrate an existing on prem VM running on KVM hypervisor to GCP
+  hosts: localhost
+  gather_facts: false
+
+  vars_files:
+    - vars/main.yml
+
+  tasks:
+    - name: Add host to inventory
+      ansible.builtin.add_host:
+        name: "{{ kvm_host.name }}"
+        ansible_host: "{{ kvm_host.ansible_host }}"
+        ansible_user: "{{ kvm_host.ansible_user }}"
+        ansible_ssh_common_args: -o "UserKnownHostsFile=/dev/null" -o StrictHostKeyChecking=no -i {{ kvm_host.ansible_ssh_private_key_file }}
+        groups: "{{ kvm_host.groups }}"
+
+    - name: Import 'cloud.gcp_ops.clone_on_prem_vm' role
+      ansible.builtin.import_role:
+        name: cloud.gcp_ops.clone_on_prem_vm
+      vars:
+        clone_on_prem_vm_source_vm_name: "{{ clone_on_prem_vm_source_vm_name }}"
+        clone_on_prem_vm_image_name: "{{ clone_on_prem_vm_image_name }}"
+        clone_on_prem_vm_local_image_path: "{{ clone_on_prem_vm_local_image_path }}"
+        clone_on_prem_vm_overwrite: "{{ clone_on_prem_vm_overwrite }}"
```

Comment:
Should we add the `clone_on_prem_vm_uri` var too?

Reply:
Added, thanks!